packages/electron/src/electron/main/index.ts

```diff
@@ -139,6 +141,9 @@ app.on("window-all-closed", () => {
 
 const isMac = process.platform === "darwin";
 
+// @ts-expect-error updateInfo is initialized in the app.on("ready") event
```

Comment:
We should not be suppressing this error and fix it instead. See my comment above.